### DISCUSSION TITLE
how to disable the virtual text in nvim with ccls?

### DISCUSSION THREAD

**Discussion by shun-li-casia** [neoclide/coc.nvim#4729] [Q&A] [answered]
hello guys!

I am using ccls with coc.nvim. I found the virtual text generated by the ccls is very annoying, is there any tips that can disable these messages?

![image](https://github.com/neoclide/coc.nvim/assets/139104710/c7ef11ac-9917-4db8-b4f4-faddd76db093)

**Comment by fannheyward** [comment, marked as answer]
Set `codeLens.enable` to `false` to disable it.

**Reply by lee-shun** [reply]
thanks for quick reply, I will check it later
🤟🤟

**Reply by chemzqm** [reply]
`:h coc-code-lens`
